lib/hanami/generators/app/application.rb.tt

```diff
@@ -47,22 +49,29 @@ module <%= config[:classified_app_name] %>
       # host 'example.org'
 
       # URI port used by the routing system to generate absolute URLs
-      # Argument: An object coercible to integer, default to 80 if the scheme is http and 443 if it's https
-      # This SHOULD be configured only in case the application listens to that non standard ports
+      # Argument: An object coercible to integer, default to 80 if the scheme
+      # is http and 443 if it's https
+      #
+      # This SHOULD be configured only in case the application listens to that
+      # non standard ports
       #
       # port 443
 
       # Enable cookies
       # Argument: boolean to toggle the feature
       #           A Hash with options
       #
-      # Options: :domain   - The domain (String - nil by default, not required)
-      #          :path     - Restrict cookies to a relative URI (String - nil by default)
-      #          :max_age  - Cookies expiration expressed in seconds (Integer - nil by default)
-      #          :secure   - Restrict cookies to secure connections
-      #                      (Boolean - Automatically set on true if currently using a secure connection)
-      #                      See #scheme and #ssl?
-      #          :httponly - Prevent JavaScript access (Boolean - true by default)
+      # Options:
+      #   :domain   - The domain (String - nil by default, not required)
+      #   :path     - Restrict cookies to a relative URI
+      #               (String - nil by default)
+      #   :max_age  - Cookies expiration expressed in seconds
+      #               (Integer - nil by default)
+      #   :secure   - Restrict cookies to secure connections
+      #               (Boolean - Automatically set on true if currently using a
+      #               secure connection)
```

Comment:
71 and 72 can be combined into one line by rewording to:
`     #               (Boolean - Automatically true when using HTTPS)`